packages/atproto_firehose/client.py

```diff
@@ -124,9 +124,18 @@ def _websocket_uri(self) -> str:
         return _build_websocket_uri(self._method, self._base_uri, self._params)
 
     def _get_client(self) -> 'SyncWebSocketClient':
-        return connect(self._websocket_uri, max_size=_MAX_MESSAGE_SIZE_BYTES, close_timeout=0.1)
+        # Disable automatic pings for sync client (added in websockets v15+)
+        # to maintain behavior consistent with <v14 for now.
+        return connect(
+            self._websocket_uri,
+            max_size=_MAX_MESSAGE_SIZE_BYTES,
+            close_timeout=0.1,
+            # see https://websockets.readthedocs.io/en/stable/topics/keepalive.html
+            ping_interval=None,  # Disable automatic pings
+        )
```

Comment:
the important part of the diff

Reply:
yeah i didn't encounter any issues with them, i was just erring toward caution based on my reading of the original issue, but I can go ahead and test it for a longer period of time with it on